addons/binding/org.openhab.binding.freebox/README.md

```diff
@@ -0,0 +1,63 @@
+# Freebox Binding Configuration
+
+This binding integrates the [Freebox Revolution](http://www.free.fr/adsl/freebox-revolution.html) to your openHab installation.
+
+![Freebox Revolution](doc/freebox-revolution.jpg)

```

Comment:
For further use, but you're right, I picked up these pictures on the net and rights are not clear, so I remove them.